ibis/omniscidb/client.py

```diff
@@ -488,6 +488,62 @@ def alter(self, tbl_properties=None):
         """
         raise NotImplementedError('This method is not implemented yet!')
 
+    def insert(
+        self,
+        obj: Optional[Union[pd.DataFrame, ir.Expr]] = None,
+        dst_cols: Optional[list] = None,
+        values: Optional[list] = None,
+        validate: Optional[bool] = True,
+    ):
+        """
+        Insert data into OmniSciDB table.
+
+        Parameters
+        ----------
+        obj : pd.DataFrame or ibis Expr, optional
+            Set obj to pandas dataframe or ibis Expr. Default is None
+        dst_cols : list, optional
+            Set list of table's column(s) into which data will be coppied
+            when ibis Expr or values is passed. Default is None
+        values : list, optional
+            Set list of values. Default is None
+        validate : boolean, optional
+            If True, do more rigorous validation that schema of table being
+            inserted is compatible with the existing table. Default is True
+
+        Examples
+        --------
+        >>> table_name_1 = 'my_table1'
+        >>> my_table1 = con.table(table_name_1)  # doctest: +SKIP
+        >>> import pandas as pd
+        >>> data = {'a': [1], 'b': [4.0]}
+        >>> df = pd.DataFrame(data=data)
+        >>> my_table1.insert(df)  # doctest: +SKIP
+        >>> my_table1 = con.table(table_name_1)  # doctest: +SKIP
+        >>> table_name_2 = 'my_table2'
+        >>> my_table2 = con.table(table_name_2)  # doctest: +SKIP
+        >>> dst_cols2 = ['c', 'd']
+        >>> my_table2.insert(
+        ...     my_table1['a', 'b'], dst_cols2
+        ... )  # doctest: +SKIP
```

Comment:
Can you clean up the examples? I think it's overcomplicated. And it's probably better to divide in two different examples, and add comments to let users know what is going on.